src/lib.rs

```diff
@@ -484,6 +485,11 @@ note: skipped the following crates since they have no library target: {skipped}"
                     );
                 }
 
+                let workspace_overrides =
+                    manifest::deserialize_lint_table(metadata.workspace_metadata.clone())
+                        .context("[workspace.metadata.cargo-semver-checks] table is incorrect")?
```

Comment:
nit: word choice, here and below as well
```suggestion
                        .context("[workspace.metadata.cargo-semver-checks] table is invalid")?
```